english/use-the-fork-luke/use-the-fork.md

```diff
@@ -1,5 +1,6 @@
 The hand of marshmallow has left you with so many questions. What did it want with the lighter?
-What does it want with me? Why did it have a ring-finger tan line?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.